Updating translation files

RyanNoelk opened the pull request:
I'm using a script to auto-update the translation files ([found here](https://github.com/RyanNoelk/OpenEats/blob/master/docs/React_i18n.md)). It decided to move a few of the translations around. I'm going to go though it here on GitHub and point out any discrepancies as well as any new strings that need translating.

Please point out any issues found.

RyanNoelk committed:
updating translation files

RyanNoelk committed:
updating some german translations

RyanNoelk committed:
hide supported sites

RyanNoelk commented on frontend/locale/es.json:
```diff
@@ -9,28 +9,68 @@
+  "new_list.header": "Crear una nueva lista",
+  "new_list.placeholder": "Dele un nombre a la lista",
+  "new_list.button": "Crear mi lista",
+  "news.browse_recipe_button": "Browse All Recipes",
```
Translation needed

RyanNoelk commented on frontend/locale/es.json:
```diff
@@ -9,28 +9,68 @@
   "autocomplete.no_matches": "No hay resultados para {value}",
+  "directions.next_direction": "Próxima instrucción",
+  "ingredients.add_ingredient": "Añadir ingrediente",
+  "subrecipe.add_link": "Add Link",
```
Translation needed

RyanNoelk commented on frontend/locale/es.json:
```diff
@@ -9,28 +9,68 @@
+  "recipe.import.error_message_2": "This Website is not supported.",
+  "recipe.import.error_message_3": "Please enter a valid URL.",
+  "recipe.import.hide_sites": "hide supported sites",
+  "recipe.import.show_sites": "show supported sites",
```
Translation needed for all `recipe.import` values

RyanNoelk commented on frontend/locale/es.json:
```diff
@@ -44,51 +84,24 @@
   "recipe.create.servings_placeholder": "Sirve",
   "recipe.create.rating_label": "Estrellas",
   "recipe.create.rating_placeholder": "Dele una valoración del 0 a 5",
+  "recipe.create.subrecipes_label": "Recipe links",
```
translation needed

RyanNoelk commented on frontend/locale/de.json:
```diff
@@ -9,28 +9,68 @@
+  "recipe.import.error_message_2": "This Website is not supported.",
+  "recipe.import.error_message_3": "Please enter a valid URL.",
+  "recipe.import.hide_sites": "hide supported sites",
+  "recipe.import.show_sites": "show supported sites",
```
translation needed for all `recipe.import` values

DJCrashdummy committed:
fix/i18n (#255)

* "Open Eats" --> "OpenEats"

* "Open Eats" --> "OpenEats"

* "Open Eats" --> "OpenEats"

* corrected some upper-/lower-case

DJCrashdummy committed:
fix/i18n-de (#256)

* fixed translation in wrong line

* polished the existent translations

* updated german translation

translated all recipe.import, subrecipe* values and and ingredients.group_placeholder.

* Tag-translation

either translate "tag" everywhere or nowhere...

DJCrashdummy commented on frontend/locale/en.json:
```diff
@@ -9,28 +9,68 @@
+  "nav.recipes": "Browse recipes",
+  "grocery_list.my_lists": "My Lists",
+  "grocery_list.footer": "Double Click to edit an item.",
+  "list.new-input-placeholder": "What do you need to buy?",
```
because of a not really appropriate german translation (can be translated back into something like "What do you need?"), i thought a little bit more about **"list.new-input-placeholder"**: why has everything to be bought? - i know it is a grocery list, but what if you have a little garden or something like that...?
what do you think about a more general wording like _"What is still left?"_, _"What lacks on the list?"_, _"Is something still missing?"_ or a mixture of them?

RyanNoelk replied:
maybe:
- What else do you need?
- What do you need?

DJCrashdummy replied:
well... although the translations (at least the english & german one) now match correct, it is an ungainly solution and from my point of view it got worse. :-(
maybe i dislike this wording because "_What do you need?_" (= "_Was brauchen/brauchst Sie/Du?_") is used by drug-dealers to start a transaction... so our grocery list sounds like a very polite drug-dealer. `*lol*`

i go with my suggestions: (now prioritized)
```
What is (still) missing (on the list)?	= Was fehlt (noch) (auf der Liste)?
What is (still) left (on the list)?	= Was fehlt (noch) (auf der Liste)?
What is (still) lacking on the list?	= Was fehlt (noch) auf der Liste?
What lacks on the list?			= Was fehlt auf der Liste?
What else do you need?			= Was benötigen Sie noch?
Is something still missing?		= Fehlt noch etwas?
```

RyanNoelk replied:
My favorite is `What else do you need?` and just dropping the `to buy`.

DJCrashdummy replied:
you are the boss... i'm ok with it.

DJCrashdummy commented on frontend/locale/en.json:
```diff
@@ -9,28 +9,68 @@
   "404.message": "Sorry the page came back with a 404 error we can't find what you are looking for.",
   "footer.credit": "Created with {link}",
   "footer.icon_credit": "Icons made by {link} ({ccLink})",
```
1. i don't know where to correct it, but i want to mention it:
the text of the **{ccLink}** is wrong, the license is abbreviated as "**(CC BY 3.0)**".

2. why make the whole thing longer than necessary - it becomes problematic on smaller screens (especially with longer translations):
wouldn't _"Icons by {link} ({ccLink})"_ be enough?

RyanNoelk replied:
Both of your points are correct. I'll make the changes.

RyanNoelk committed:
fixing some english translations per code review

DJCrashdummy committed:
Fix/cc-link (#257)

* corrected minor mistake

- no need for doubling the brackets
- using the tool-tip for exact the same does not make much sense...: i liked at least the explanation of "Creative Commons"

* Update de.json

* Update es.json

DJCrashdummy committed:
minor translation-update (#260)

* updated de-string according to dicussion

https://github.com/RyanNoelk/OpenEats/pull/254/files/11781ba19e3180cfdb4db32a1520f88068f6f941#r136663191

* updated en-string according to discussion

https://github.com/RyanNoelk/OpenEats/pull/254/files/11781ba19e3180cfdb4db32a1520f88068f6f941#r136663191

bglopez committed:
Update Spanish translation (#259)

RyanNoelk committed:
removing space in OpenEats

RyanNoelk committed:
adding missing spanish translations (provided by google)

RyanNoelk committed:
Merge branch 'dev' into feature/i18n-udpate

RyanNoelk commented:
I'm going to merge this now as we have made very good progress with it. If any other changes need to be made, please either open an issue or create a PR.
